mysql/datadog_checks/mysql/queries.py

```diff
@@ -74,6 +74,87 @@
 SELECT plugin_status
 FROM information_schema.plugins WHERE plugin_name='group_replication'"""
 
+SQL_DATABASES = """
+SELECT schema_name as `name`,default_character_set_name,default_collation_name FROM information_schema.SCHEMATA
+WHERE schema_name not in ('mysql', 'performance_schema', 'information_schema')"""
+
+SQL_TABLES = """\
+SELECT table_name as `name` FROM information_schema.TABLES WHERE TABLE_SCHEMA = %s
+"""
```

Comment:
please let me know if I omitted to many columns 
https://dev.mysql.com/doc/refman/8.4/en/information-schema-tables-table.html

Reply:
Oh I did not realize how rich this table was with data. I see some valuable ones here:

- `ENGINE` - nowadays InnoDB is the most common by far, but the quality of our insights depends on us knowing this
- `ROW_FORMAT`
- `CREATE_TIME` - very good to know and should remain static

Lots of other valuable data, but it's dynamic and better served with metrics than capturing in schemas.

Reply:
great, thx, will add.